packages/menu-bar/test/dom/menu-bar.test.js

```diff
@@ -50,4 +55,15 @@ describe('menu-bar', () => {
     await nextRender();
     await expect(menu._subMenu.$.overlay).dom.to.equalSnapshot(SNAPSHOT_CONFIG);
   });
+
+  it('overflow', async () => {
+    menu.style.width = '50px'; // hide all buttons but the "More options" button
+    await nextRender();
+    menu._overflow.click(); // show the overlay menu (More options)
+    menu._overflow.click(); // to hide the overlay menu again
+    menu.style.width = null; // to restore the original size of menubar
+    await nextRender();
+    await nextRender();
```

Comment:
I don't like this... smells flaky to me. Not sure why, but I have to call nextRender twice, otherwise the menubar is not in the state I want it to be (it is still hiding the items after first nextRender) - is there other way how to wait for the visibility of an element?

Reply:
I think this is a result of `menu-bar` using a `ResizeObserver` to get notified of width changes.

Here is an example of how we usually wait for resize observers in tests:
https://github.com/vaadin/web-components/blob/25db6771b3128cce504ba9bdf7c69cc2d8f0dd30/packages/board/test/helpers.js#L6-L21